cmake/Caffe2Config.cmake.in

```diff
@@ -86,11 +86,7 @@ if(@USE_CUDA@)
   set(CAFFE2_USE_CUDA @USE_CUDA@)
   set(CAFFE2_USE_TENSORRT @USE_TENSORRT@)
 
-  # Add current directory to module path so we pick up FindCUDAToolkit.cmake
-  set(old_CMAKE_MODULE_PATH CMAKE_MODULE_PATH)
-  list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
   include("${CMAKE_CURRENT_LIST_DIR}/public/cuda.cmake")
-  set(CMAKE_MODULE_PATH old_CMAKE_MODULE_PATH)
```

Comment:
Why remove this?

Reply:
It causes CMAKE_MODULE_PATH set to string “old_CMAKE_MODULE_PATH” and because now I always load our version of findcudatoolkit

Reply:
Good point, there is a bug, but why are we letting our internal changes to `CMAKE_MODULE_PATH` escape into the users environment?

Reply:
That makes sense. I add them back.